mailpoet/package.json

```diff
@@ -158,12 +158,13 @@
     "@types/wordpress__url": "^2.14.0",
     "@wordpress/browserslist-config": "^4.1.2",
     "@wordpress/dependency-extraction-webpack-plugin": "^3.4.1",
-    "@wordpress/scripts": "^22.3.0",
+    "@wordpress/scripts": "^23.3.0",
```

Comment:
I saw that you mentioned in the commit message that due to how we use this package, it is safe to update it here. That being said, the guide for updating @wordpress packages mentions that "Since we experienced that upgrading packages one by one may lead to ending up with multiple versions of different JS packages the recommended approach is removing all @wordpress/* packages and installing them again."

So could you please share more details on why we need to update this package? Maybe since it is a dev dependency it is fine to update it in another ticket together with the rest of the @wordpress packages?

Reply:
I guess in this case it only makes sense to update other packages starting with `@wordpress/`  in devDependencies, what do you think?

Reply:
I don't know to be honest, but I would assume it would be necessary to update all `@wordpress/` and not only those in `devDependencies`.

If there are no pressing reasons to update `@wordpress/scripts`, which seems to me to be the case, but please correct me if I'm wrong, I would suggest waiting and updating `@wordpress/scripts` the next time someone works on the ticket to update the `@wordpress/` packages instead of updating it in this PR.